src/EditorFeatures/CSharpTest/Diagnostics/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -192,7 +192,7 @@ public void TestConflictingProperties()
         {
             Test(
 @"class Test : [|I1|] { int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ",
-@"using System ; class Test : I1 { int I1 . Prop { get { throw new NotImplementedException ( ) ; } set { throw new NotImplementedException ( ) ; } } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
+@"class Test : I1 { int I1 . Prop { get ; set ; } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
```

Comment:
I'm not sure -- though we should discuss this sort of design change before making them. I'm not a huge fan of dramatically changing how features work in an update.

@jmarolf, could you add context from the original bug so we know why this change is necessary? Better yet, could you port the relevant portion of the bug to GitHub?